pkg/traceutil/trace.go

```diff
@@ -0,0 +1,60 @@
+package traceutil
```

Comment:
`pkg/util/trace` instead?  Can still import alias it to traceutil..

Reply:
good point!

Reply:
'pkg/' is a collection of utility packages in etcd. I followed the naming convention like other utility packages.

Reply:
Sounds good.

Reply:
Ah, the structure has changed. It used to be that we had a bunch of utils under `pkg/util`. Given the new structure, `traceutil` make sense.